docs/blogs/zoda.md

```diff
@@ -0,0 +1,376 @@
+---
+title: "Optimizing Block Dispersal with Coding"
+description: "In which we describe how we use coding schemes, and ZODA to distribute blocks efficiently"
+date: "October 15th, 2025"
+author: "Lucas Meier"
+author_twitter: "https://x.com/cronokirby"
+katex: true
+---
+
+A crucial part of blockchain consensus is distributing the block among participants.
+You can come to consensus over a mere fingerprint of the block---a hash for example---
+but doing anything interesting with the block, like processing transactions, requires
+the data.
+
+We've invested some effort in this part of the Commonware Library recently, and I'd
+like to share some fruits of that effort in this post.
+As an outline, we'll cover:
+
+- data dissemination, naively,
+- how to increase efficiency with coding,
+- and, finally, how to get quicker guarantees about the data with [ZODA](https://eprint.iacr.org/2025/034).
```

Comment:
In the intro here it might be good to provide a bit more detail about what ZODA gives us that more typical coding schemes don't, since it takes a good bit of reading to get to that point below.  I worry the reader might assume we're just doing something RS+KZG like and miss the coolness that ZODA brings.

Reply:
Added a small paragraph about this